static/src/stylesheets/module/onward/_feedback.scss

```diff
@@ -0,0 +1,55 @@
+
+.feedback__form {
+    border: 0px;
+    outline: none;
+    display: none;
+}
+
+.feedback__form--selected {
+    display: block;
+}
+
+.feedback__element {
+    margin-bottom: 1.5rem;
+    margin-top: 1.5rem;
+}
+
+.feedback__entry {
+    border: 1px solid #dcdcdc;
+    padding: .75rem;
+    outline: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border-radius: .5rem;
+    display: block;
+    width: 80%;
+}
+
+.feedback__textarea {
+    height: 13em;
+}
+
+.feedback__select {
+    border: 1px solid #dcdcdc;
```

Comment:
Most of our colours are defined as variables in [pasteup](http://guardian.github.io/pasteup-palette/demo/)

You can replace `#dcdcdc` with `$neutral-4`